kyco/core/switch.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 *-*
+"""Module with main classes related to Switches"""
+import logging
+
+from kyco.core.events import KycoSwitchDown
+from kyco.core.events import KycoSwitchUp
+
+log = logging.getLogger('Kyco')
+
+
+class KycoSwitch(object):
+    """This is the main class related to Switches modeled on Kyco."""
+
+    def __init__(self, switch_id, socket=None):
+        self.switch_id = switch_id
```

Comment:
@beraldoleal considering that we are going to use the dpid as the identifier of the switch, wouldn't be better if we use the dpid on the events instead of `connection` (`(ip, port)`) ?
I mean, we don't want to know the "connection" on the events, what we really need is a reference to the switch. right?